PathKit/Sources/PathKit/Point.swift

```diff
@@ -46,6 +46,35 @@ extension Point {
         self.x = Double(x)
         self.y = Double(y)
     }
+
+    /// Ceeates a point located at the head of `vector`.
+    ///
+    /// - Parameter vector: The vector used to create the point.
+    public init(_ vector: Vector) {
```

Comment:
Not a fan of this.

Reply:
Would you just completely remove this?

Reply:
I would give it a better name. For instance `Point.origin.displace(by: vector)`

Reply:
I guess we can just do `.zero + vector`